docs/reference/ml/anomaly-detection/aggregations.asciidoc

```diff
@@ -25,7 +29,14 @@ You must also consider the interval of the date histogram aggregation carefully.
 The bucket span of your {anomaly-job} must be divisible by the value of the
 `calendar_interval` or `fixed_interval` in your aggregation (with no remainder).
 If you specify a `frequency` for your {dfeed}, it must also be divisible by this
-interval.
+interval. You cannot feed an {anomaly-job} with a date histogram with an 
```

Comment:
> You cannot feed an anomaly detection job with a date histogram...

This working seems strange to me.  We don't talk about "feeding jobs" elsewhere AFAIK. Could this be changed to something like this?: "Anomaly detection jobs cannot use date histograms with...." or "Data feeds cannot use date historgrams with..."